.github/workflows/build.yml

```diff
@@ -46,6 +48,8 @@ jobs:
       - name: Setup MSVC
         uses: egor-tensin/vs-shell@9a932a62d05192eae18ca370155cf877eecc2202 # v2.1
         if: ${{ matrix.target_os == 'windows' }}
+      - name: install python3-requests
```

Comment:
I tought this was added to dockers itself, do we need it here?

Reply:
Sames as https://github.com/NordSecurity/libtelio/pull/543#discussion_r1608039724